app/src/main/java/com/willowtree/vocable/utils/VocableSharedPreferences.kt

```diff
@@ -16,6 +16,8 @@ class VocableSharedPreferences : KoinComponent {
         const val KEY_HEAD_TRACKING_ENABLED = "KEY_HEAD_TRACKING_ENABLED"
         const val KEY_SENSITIVITY = "KEY_SENSITIVITY"
         const val DEFAULT_SENSITIVITY = 0.1F
+        const val DWELL_TIME = "DWELL_TIME"
+        const val DEFAULT_DWELL_TIME = 1000L
```

Comment:
We should only define these defaults in one place. Probably best to use the ones in the fragment.
`private const val DEFAULT_SENSITIVITY = SensitivityFragment.MEDIUM_SENSITIVITY`
`private const val DEFAULT_DWELL_TIME = SensitivityFragment. DWELL_TIME_ONE_SECOND`